tests/src/test/scala/pathy/path.scala

```diff
@@ -100,133 +84,194 @@ class PathSpecs extends Specification with ScalaCheck {
     ).toList must_== List("c", "foo", "bar", "flat.md")
   }
 
-  "parseRelFile - image.png" in {
-    parseRelFile("image.png") must beSome(file("image.png"))
-  }
+  "parseRelFile" >> {
```

Comment:
As a general comment, I think it would be good to make it clearer that all of these tests apply to `posixCodex` specifically. How about this?
```scala
"posixCodec" >> {
  import posixCodec._

  "parseRelFile" >> {
    "should X" in {
```